include/zephyr/bluetooth/conn.h

```diff
@@ -323,6 +323,32 @@ enum bt_conn_state {
 	BT_CONN_STATE_DISCONNECTING,
 };
 
+/** Security level. */
+typedef enum __packed {
+	/** Level 0: Only for BR/EDR special cases, like SDP */
+	BT_SECURITY_L0,
+	/** Level 1: No encryption and no authentication. */
+	BT_SECURITY_L1,
+	/** Level 2: Encryption and no authentication (no MITM). */
+	BT_SECURITY_L2,
+	/** Level 3: Encryption and authentication (MITM). */
+	BT_SECURITY_L3,
+	/** Level 4: Authenticated Secure Connections and 128-bit key. */
+	BT_SECURITY_L4,
+	/** Bit to force new pairing procedure, bit-wise OR with requested
+	 *  security level.
+	 */
+	BT_SECURITY_FORCE_PAIR = BIT(7),
+} bt_security_t;
+
+/** Connection Info Flags. */
+enum __packed {
+	/** Paired with Secure Connections. */
+	BT_CONN_FLAG_SC,
+	/** Paired with Out of Band method. */
+	BT_CONN_FLAG_OOB,
```

Comment:
I do wonder that, since we've already added `BT_SECURITY_FORCE_PAIR` to `bt_security_t`, whether we should just expand the `bt_security_t` with the OOB and SC flags?

Reply:
I would prefer not to mix those